HyperV/scripts/create-environment.ps1

```diff
@@ -211,29 +213,36 @@ function cherry_pick($commit) {
 }
 
 ExecRetry {
-    pushd C:\OpenStack\build\openstack\neutron
-    & pip install C:\OpenStack\build\openstack\neutron
+    ls ( Get-Item $buildDir ).Parent.FullName
```

Comment:
shouldn't this be before the ExecRetry? (ExecRetry section should be almost identical for all "projects", no?)